src/Glpi/Form/Export/Serializer/FormSerializer.php

```diff
@@ -82,6 +83,7 @@ public function exportFormsToJson(array $forms): ExportResult
 
     public function previewImport(
         string $json,
+        array $skipped_forms,
```

Comment:
I think it make sense that the import can be run without ignoring any form, thus maybe this parameter should be optional (move it to last + add `[]` default value) ?